assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-runtime-rhel.json

```diff
@@ -35,7 +75,7 @@
 			{
 				"name": "3.1-ubi8",
```

Comment:
3.1-ubi8 tag is not removed from this pr

3.1-ubi8
  tagged from registry.access.redhat.com/ubi8/dotnet-31:3.1